apidoc/Titanium/UI/2DMatrix.yml

```diff
@@ -291,7 +291,7 @@ properties:
         Point to rotate around, specified as a dictionary object with `x` and `y`
         properties, where { x: 0.5, y: 0.5 } represents the center of whatever is being
         rotated.
-    type: Dictionary
+    type: Point
```

Comment:
Also should be changed in Matrix2D.

Reply:
good catch, just fixed this in next commit